### DISCUSSION TITLE
Can I include language data from fork in Language Card?

### DISCUSSION THREAD

**Discussion by prem-k-r** [anuraghazra/github-readme-stats#4025] [Q&A]
Compact Language Card Layout

**Comment by MrSuddenJoy** [comment]
MIT, so do whatever.

**Reply by prem-k-r** [reply]
Ah, I didn't mean that. 
In Language Card, data from only your personal repo is considered. Is there a way we can include data which is not personal repo like a fork of a public repo.

**Reply by MrSuddenJoy** [reply]
@prem-k-r To be completely I dont think so.

**Reply by ilianoKokoro** [reply]
> Ah, I didn't mean that. In Language Card, data from only your personal repo is considered. Is there a way we can include data which is not personal repo like a fork of a public repo.

@prem-k-r 
From what I understand, contributions to forks is not included in the data because it is owned by another user. It can't be added.
